src/test/functional/steps/opening-hours.ts

```diff
@@ -87,6 +87,19 @@ When('I enter an incomplete opening hour description', async () => {
   await I.setElementValueAtIndex('#openingTimesTab input[name$="[hours]"', entryFormIdx, '10:00am to 4:00pm', 'input');
 });
 
+When('I left the opening hours blank and select description at index {int}', async (type_id: number) => {
```

Comment:
Our linter is set to use camelcase for variables, so you'll need to change this from snake_case to camelCase, like below :)
```suggestion
When('I left the opening hours blank and select description at index {int}', async (typeId: number) => {
```